src/Uno.UI/UI/Xaml/ElementStub.cs

```diff
@@ -21,6 +24,32 @@ namespace Windows.UI.Xaml
 	/// <remarks>This control is added in the visual tree, in place of the original content.</remarks>
 	public partial class ElementStub : FrameworkElement
```

Comment:
I'm curious, do we have any mechanism for this stub to be hidden from consumer code - eg when enumerating Panel.Children, traversing the visual tree etc?

Reply:
Not really, and neither does UWP iirc. Do you thinkg of a specific scenario where that could cause issues ?

Reply:
Does UWP put an ElementStub-type placeholder? I always assumed this was an Uno implementation detail.

No specific scenario that I know of, just curious.

Reply:
There's no such ElementStub, the WinUI parser handles this differently because of the way it is materialized. We may end up there as well in the future.